getting_started/build-guide/build-from-scratch.rst

```diff
@@ -311,40 +335,40 @@ build with -j [n]
    ./scripts/xtensa-build-all.sh -d -r apl
    ./scripts/xtensa-build-all.sh -d -r -j 4 apl
 
-Build with commands
--------------------
-
-This is a detailed build guide for the *sof* repo.
+Incremental builds
+------------------
 
-Snippets below assume that your working directory is repo's root
-(~/work/sof/sof/).
+This is a more detailed build guide for the *sof* repo. Unlike
+``xtensa-build-all.sh``, this doesn't rebuild everything every time.
 
-CMake is designed for out-of-tree builds which is why you should make separate
-dirs for your configurations.
+Snippets below assume that your current directory is the root of the
+``sof`` clone (``~/work/sof/sof/``).
 
-You can manage builds for many configurations/platforms from the one source
-this way.
+CMake recommends out-of-tree builds. Among others, this lets you build
+different configurations/platforms in different build directories from
+the same source without starting from scratch.
 
 .. note::
 
-   The *-j* argument indicates the number of cores to use in the build
-   process. Select a value that matches your build system.
+   The ``-j`` argument tells make how many processes to use concurrently.
+   Select a value that matches your build system.
 
 for |BYT|:
 
 .. code-block:: bash
 
    mkdir build_byt && cd build_byt
-   cmake -DTOOLCHAIN=xtensa-byt-elf -DROOT_DIR=`pwd`/../../xtensa-root/xtensa-byt-elf ..
+   cmake -DTOOLCHAIN=xtensa-byt-elf -DROOT_DIR="$XTENSA_ROOT"/xtensa-byt-elf ..
+   make help # lists all available targets
    make baytrail_defconfig
-   make bin -j4
+   make bin -j4 VERBOSE=1
```

Comment:
can we make the verbose to be optional?

Reply:
So this just an example, so anyone can leave it out.
On the other hand, someone not familiar with `cmake` would not be able to guess it.
See commit message.